.travis.yml

```diff
@@ -60,40 +63,51 @@ addons:
       - cmake-data
 
 install:
-    - if [[ "$CC" == "gcc" ]]; then export CXX="g++-4.8"; fi
-    - cmake --version
-    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-        export CONDA_OS=MacOSX;
-      else
-        export CONDA_OS=Linux;
-      fi
-    - if [[ $PYTHON_VERSION == 2.7 ]]; then
-        export TRAVIS_PYTHON_VERSION="2.7";
-      else
-        export TRAVIS_PYTHON_VERSION="3.6";
-      fi
-    # We do this conditionally because it saves us some downloading if the version is the same.
-    - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-        wget https://repo.continuum.io/miniconda/Miniconda2-latest-${CONDA_OS}-x86_64.sh -O miniconda.sh;
-      else
-        wget https://repo.continuum.io/miniconda/Miniconda3-4.4.10-${CONDA_OS}-x86_64.sh -O miniconda.sh;
-      fi
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export CONDA_HOME="$HOME/miniconda"
-    - export PATH="$CONDA_HOME/bin:$PATH"
-    - pip install -r requirements.txt
-    - hash -r
-    - conda config --set always_yes yes --set changeps1 no
-    - conda update -q conda
-    # Useful for debugging any issues with conda
-    - conda info -a
-    - wget https://raw.githubusercontent.com/Statoil/ert/master/travis/install_python_packages.py
-    - python install_python_packages.py
+  - if [[ "$CC" == "gcc" ]]; then export CXX="g++-4.8"; fi
+  - cmake --version
+  - if [[ $PYTHON_VERSION == 2.7 ]]; then
```

Comment:
I think you can remove all of the `TRAVIS_PYTHON_VERSION` :)

Reply:
Yeppsi! Fixed :)